### DISCUSSION TITLE
Allow password reset for specific users

### DISCUSSION THREAD

**Discussion by Autive** [snicco/fortress#5] [Feature Requests]
Sometimes we got a client that has (some) admin rights and we need him/her to be able to reset their password (temporary).

Best possible feature: Having a CLI command where we can allow an one time allow reset password for a specific user, disregarding any possible configuration.

If that's not possible, allow a specific user (by email or username) by config to reset their password.

**Comment by alpha-pix** [comment]
Yes, it is important for the user to self-serve reset because otherwise there are issues getting the new pw to them and into their password manager. Maybe there could be a CLI command to allow resets temporarily eg. for 30m for specified user.

**Comment by calvinalkan** [comment]
Both good ideas,

I think you mean like:

...`fortress password reset admin --link ---email`

**Reply by alpha-pix** [reply]
Yes, maybe with an default time of 30 mins and optional time modifier. Sure there will be times when you have to do it twice because the user didn't get around to it, but it is better to create urgency and avoid the risk that the user won't get round to it for a couple of days.

**Comment by alpha-pix** [comment]
Also, you can temporarily reduce their permissions so they can reset. If they want the permissions back they can confirm they have reset it.

**Reply by calvinalkan** [reply]
This is actually a brilliant idea ;)

You can then even use the WP Admin "send this user a password link" flow.
